### ISSUE TITLE
Expressionproperties change model through observable

### ISSUE THREAD

**Issue by GillesVercammen** [ngx-formly/ngx-formly#2094]
Hello,

I see in the interface that expressionproperties take either a string, callback or an observable.
when using expressionproperties to change templateOptions i can succesfully provide an observable and it works. 
But when i try to change the model through expressionproperties, i can't provide an observable. 
Is this intended?

For example:

```
expressionProperties: {
        'model.text': of("test"),
        'templateOptions.label': of('labelObs'),
}

```
the 'templateOptions.label' does show the label correctly as labelObs.
The 'model.text' does not show 'test' as value inside the input..

i can only effectively change the model via expressionproperties by providing a callback like so;
`'model.text': ( ) => 'test'`

so is this the only options to change the model through expressionproperties?

Tnx!

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.5.12](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.5.12) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Comment by GillesVercammen** [comment]
many thanks for the quick response! This will help us greatly in our project
